Automattic/wp-calypso
# Disable site accelerator picker on private atomic sites

#### Changes proposed in this Pull Request

Both Photon and Photon CDN are not available when the site is atomic and private. This PR reflects this fact on `/settings/performance/` page:

**Unlaunched private site:**
<img width="753" alt="Zrzut ekranu 2020-02-26 o 16 04 09" src="https://user-images.githubusercontent.com/205419/75357417-a5108380-58b1-11ea-8f87-4b42444b0fdf.png">

**Launched private site:**
<img width="747" alt="Zrzut ekranu 2020-02-26 o 15 17 39" src="https://user-images.githubusercontent.com/205419/75353225-5829ae80-58ab-11ea-9baf-d142e4829f11.png">

#### Testing instructions

1. Test locally, this won't work on calypso.live.
1. Create a new site and go atomic - it's going to be private by default.
1. Don't launch it.
1. Go to `/settings/performance/` and confirm it looks like on the *first* screen above.
1. Click the "Continue" button, confirm the photon is now available. 
1. Make the site private again.
1. Go to `/settings/performance/` and confirm it looks like on the *second* screen above.
1. Try with Jetpack and Simple sites. Confirm the eligibility gate is never displayed for these.
1. Follow the test plan on #39527 and confirm it keeps working
1. Create a new site, go Plugins -> Upload, confirm the eligibility gate behaves as expected
